dart/generate-website-screenshot/lib/main.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  Map<String, dynamic> data = jsonDecode(req.payload);
+
+  // Checking if url Parameter is present in the payload
+  if (!data.containsKey("url")) {
+    throw Exception("Missing parameter url.");
+  }
+
+  final url = data["url"];
+
+  // Checking if SCREENLY_API_KEY is provided in Enviornment.
+  if (req.variables['SCREENLY_API_KEY'] == null) {
+    res.json({
+      "success": false,
+      "message": "Missing Screenly API Key in environment variables",
```

Comment:
Please remove "environment"

```suggestion
      "message": "Missing Screenly API Key in variables",
```